WordPress/src/main/java/org/wordpress/android/ui/main/WPMainNavigationView.kt

```diff
@@ -46,6 +47,7 @@ class WPMainNavigationView @JvmOverloads constructor(
     private lateinit var fragmentManager: FragmentManager
     private lateinit var pageListener: OnPageListener
     private var prevPosition = -1
+    private var unselectedButtonAlpha = 0.0f
```

Comment:
Since this is just a `View` and doesn't have the lifecycle issues, wouldn't the following work:
```suggestion
    private val unselectedButtonAlpha = ResourcesCompat.getFloat(
            resources,
            R.dimen.material_emphasis_disabled
    )
```

Reply:
Yep, this will work!

Reply:
@khaykov It looks like the property is still a `var` and not a `val` after the change.

Reply:
Fixed! Sorry about that 🙇

Reply:
No worries!